endaq/calc/utils.py

```diff
@@ -206,3 +211,49 @@ def _rolling_slice_definitions(
     num = int(slice_width / spacing / 2)
 
     return indexes, slice_width, num, length
+
+
+def convert_units(
+        src: str,
+        dst: str,
```

Comment:
Minor suggestion: More specific names might be better, like `old_units` and `new_units`, or `units_in` and `units_out`. Source and destination are usually used when moving data from one location to another.

Reply:
agreed, I like `units_in` and `units_out`, I made the change - good suggestion!